ADOPTERS.md

```diff
@@ -8,3 +8,4 @@ This really helps the project to gain momentum and credibility. It's a small con
 | [Alliander](https://www.alliander.com/en/open-source/) | ospo@alliander.com | Alliander is mainly interested in the security side of the software supply chain. Alliander uses multiple tools and has evaluated even more, but currently ORT is unbeaten. ORT provides accurate and destailed scan results and allows us to define a through policy to use in evaluation. We are currently working to adopt ORT for thorough scans in addition to other tools. We're working with the community to get ORT running in GitHub Actions. |
 | [Deutsche Telekom AG](https://www.telekom.de/) | daniel.eder@magenta.at | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our [ORT fork](https://github.com/telekom/ort). |
 | [Sovereign Cloud Stack](https://scs.community/) | scs@osb-alliance.com | Currently in the process of evaluating how to generate a complete SBOM of our stack and secure the supply chain. We plan to use ORT to analyze our reference implementation as well as to add ORT to our [recommended operator's toolset](https://scs.community/about/#technological-vision). |
+| [HERE Technologies](https://here.com) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all our products, as well as to manage licensing risks. |
```

Comment:
> please add your organization's name to the alphabetically sorted table below

⬆️

Reply:
sorted the file

Reply:
Please squash the two commits into one.

Reply:
Also, please adhere to convention commit messages and briefly explain (in the title or body) *what* you updating.

Reply:
@bennati As a hint, a proper conventional commit title would for example be: "docs(ADOPTERS): Add HERE Technologies"

Reply:
done

Reply:
> @bennati As a hint, a proper conventional commit title would for example be: "docs(ADOPTERS): Add HERE Technologies"

There's no dot at the end of the title in @mnonnenmacher's example, but you managed to add one @bennati, which now fails the check...

Reply:
removed dot